app/views/users/index.html.erb

```diff
@@ -1,9 +1,10 @@
+<%= link_to "Sign out", destroy_user_session_path, method: :delete, data: { turbo_method: :delete } %>
```

Comment:
- [ OPTIONAL ] You have implemented a logout functionality using link_to which creates an anchor tag and cause the error below

![Screenshot from 2023-12-22 20-59-50](https://github.com/Uzair-Manzoor/blog-app/assets/112261237/5c276948-9d20-4a82-b860-df84557d4b52)

I have changed that to first check if the user is signed_in using the inbuilt method provided by devise and created a button which creates a form with submit button and it logs out without and issues

```ruby
<% if user_signed_in? %>
  <%= button_to "Sign out", destroy_user_session_path, method: :delete %>
<% end %>
```

Reply:
Hi @zeff96, thanks a lot for your insightful feedback. The suggestion will be implemented accordingly 👍